tests/Export/Adapters/AttributeAdapterTest.php

```diff
@@ -0,0 +1,899 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Tests\Adapters\Export;
```

Comment:
This should always correspond to the path.

This is the case for all adapter tests.
```suggestion
namespace FINDOLOGIC\FinSearch\Tests\Export\Adapters;
```